ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_service.proto

```diff
@@ -28,6 +28,7 @@ service CommandService {
   // - ``INVALID_ARGUMENT``: if the payload is malformed or is missing required fields
   // - ``RESOURCE_EXHAUSTED``: if the number of in-flight commands reached the maximum (if a limit is configured)
   // - ``UNAVAILABLE``: if the participant is not yet ready to submit commands or if the service has been shut down.
+  // - ``ABORTED``: if a key is missing or duplicated due to for example contention on resources
```

Comment:
Does anyone have any idea how to verify if I'm not missing any other service like the active contracts or transaction service?

Reply:
Not a quick one but the only one I can think about: scanning through them with the help of the docs and pondering if this fix is applicable or not.

Reply:
I went through them (and the tests as well) and see no other service that is applicable, thanks.